src/bluetooth-fw/nimble/advert.c

```diff
@@ -283,7 +283,7 @@ bool bt_driver_advert_advertising_enable(uint32_t min_interval_ms, uint32_t max_
   int rc;
   uint8_t own_addr_type;
   struct ble_gap_adv_params advp = {
-      .conn_mode = enable_scan_resp ? BLE_GAP_CONN_MODE_UND : BLE_GAP_DISC_MODE_NON,
+      .conn_mode = BLE_GAP_CONN_MODE_UND,
       .disc_mode = BLE_GAP_DISC_MODE_GEN,
```

Comment:
I think this is correct, looking at nimble/host/src/ble_gap.c:nimble/host/src/ble_gap.c.  It does not make sense for something to be connectable but not scannable, I think, and anyway, Nimble does not implement this.  In reconnect mode we might want to switch to directed advertising, but if PebbleOS doesn't do that already, that's definitely not in scope for us to implement for the NimBLE port!